openedx/core/djangoapps/user_api/accounts/serializers.py

```diff
@@ -88,6 +88,7 @@ def to_representation(self, user):
             "level_of_education": AccountLegacyProfileSerializer.convert_empty_to_None(profile.level_of_education),
             "mailing_address": profile.mailing_address,
             "requires_parental_consent": profile.requires_parental_consent(),
+            "account_privacy": UserPreference.get_value(user, 'account_privacy'),
```

Comment:
So the way to get a win on the teams API here is probably to pull all the preferences for a given user at once, and then populate the response from that single query, rather than getting preferences one at a time, but that depends on using the appropriate prefetch from the teams api, so maybe we should create a ticket for optimizing those views, and handle it over there. Opinions @nasthagiri ?